discover/faq.md

```diff
@@ -12,978 +12,157 @@ Hopefully, you'll find the answers here! If not then please don't hesitate to re
 </details>
 
 <details>
+<summary> Which blockchain network currently has the highest liquidity for Ocean?</summary>
 
-<summary>Is Swash "friendly" forking the Ocean market? They seem to have their own version of Ocean market/compute using the Swash token for transactions.</summary>
-
-Anyone can fork Ocean Market; the core team encourages this because it drives value to the Ocean ecosystem and OCEAN token. Daimler / Acentrik and deltaDAO are prime examples of forks. Swash has published data assets on Ocean Market. Details: The [Ocean Market](https://github.com/oceanprotocol/market) repo is a permissive open-source license (Apache 2). That code uses Ocean contracts deployed to Eth mainnet and elsewhere. Within hours one can fork Ocean Market code and launch their own data marketplace with their own branding. Ocean Market or the forks can use any token as a unit of exchange, OCEAN or otherwise. Unit-of-exchange is weak for value accrual. Better is *revenue* going to the protocol community; Ocean has that via whenever a datatoken is consumed, 0.1% goes to the Ocean community. Ocean Market is one of many templates for people to build Ocean-powered dApps; all templates are forkable. 
-1. Ocean [Templates](https://oceanprotocol.com/templates). 
-2. Introducing Ocean Templates [blog post](https://blog.oceanprotocol.com/introducing-ocean-templates-tools-to-build-next-generation-web3-dapps-67b66e519eb1).
-3. Ocean Token Model 2023 [blog post](https://blog.oceanprotocol.com/ocean-token-model-2023-2f306932f34a).
-  
+The Ethereum network has the highest liquidity for $OCEAN.  
 </details>
 
 <details>
+<summary>Can the Ocean tech stack be used without utilizing the OCEAN token?</summary>
 
-<summary>Is it possible to sell models on Ocean Protocol?</summary>
+All Ocean modules and components are open-source and freely available to the community. Developers can change the default currency from OCEAN to a different one for their dApp.
 
-Yes, you can sell any type of data. Start learning about us through the [documentation](https://docs.oceanprotocol.com/).
-  
 </details>
 
-
 <details>
+<summary>How does the ecosystem and the Ocean token benefit from the usage of the open-source tech stack when transactions can be paid in any currency?</summary>
 
-<summary>Is it true that Mercedes Benz has partnered with Ocean?</summary>
+You are right, all Ocean modules and components are open-source and freely available to the community. Developers can change the default currency from OCEAN to a different one for their dApp. But, there is a community fee involved in all transactions hapening in the ecosystem.
```

Comment:
Change to "For each consume transaction, the Ocean community gets a small fee. This happens whether OCEAN is used or not. [Here are details](./developers/contracts/fees#consume-aka.-order-fee)".

Why: the first two sentences are redundant with the question & answer above. The last sentence was not quite correct, and didn't give a reference with further detail(and had a spelling error).